eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/configuration/batch/jobs/BackPropagateRSJobConfiguration.java

```diff
@@ -34,11 +35,15 @@ public class BackPropagateRSJobConfiguration {
     // Can be parallelized across multiple species
     @Bean(BACK_PROPAGATE_RS_JOB)
     public Job processRemappedVariantsWithRSJob(
-            @Qualifier(BACK_PROPAGATE_RS_STEP) Step backPropagateRSStep,
+            // Back-propagate RS that were newly created in the remapped assembly
+            @Qualifier(BACK_PROPAGATE_NEW_RS_STEP) Step backPropagateNewRSStep,
+            // Back-propagate RS in the remapped assembly that were split or merged
+            @Qualifier(BACK_PROPAGATE_SPLIT_MERGED_RS_STEP) Step backPropagateSplitMergedRSStep,
```

Comment:
I think these two steps should be kept separated. Even if they are currently run together, they will be occurring at different time in the remapping processes in the future (`BACK_PROPAGATE_NEW_RS_STEP` will happen when we remapped new unclustered ssid and `BACK_PROPAGATE_SPLIT_MERGED_RS_STEP` will happen when we remapped already clustered ssid)

Reply:
> in the remapping processes in the future

I am guessing you mean in the "steady state" where we remap variants right after we receive them during submission, right? In that case, I agree that these two steps should be in two separate jobs that will be carried out at different times. BACK_PROPAGATE_RS_JOB is for carrying out back-propagation for assemblies where we have already carried out remapping and clustering.

Reply:
I guess my preference would be to keep them separated from the start and automate their execution outside of java. 
Java also introduce a false dependency between them (the first one fail the second one does not run).

Reply:
For now, I have created [two separate backpropagation jobs](https://github.com/EBIvariation/eva-accession/pull/338/commits/6b766bae9ee5133c7b5f2d98cc244b6a33124099). To minimize further tangents on job definitions, I have created a separate [ticket](https://www.ebi.ac.uk/panda/jira/browse/EVA-2845) for further refactoring changes.